src/sagemaker_xgboost_container/data_utils.py

```diff
@@ -60,6 +60,16 @@ def get_content_type(content_type_cfg_val):
         return LIBSVM
     elif content_type_cfg_val.lower() in [CSV, _content_types.CSV]:
         return CSV
+    elif _content_types.CSV in content_type_cfg_val.lower():
+        items = content_type_cfg_val.split(';')
+        label_size = [item for item in items if 'label_size' in item]
+        if label_size:
+            label_size = label_size[0].split('=')[1]
```

Comment:
Why is this only for the first label size? If we're making a generic solution, then might as well make it `any(label_size != 1 ... )`